app/models/user/permissions.rb

```diff
@@ -60,11 +59,11 @@ def member_permission_ids=(roles)
   end
 
   def member_permission_service_ids=(service_ids)
-    if service_ids.present?
-      service_ids = Array(service_ids).compact.map(&:to_i)
+    if service_ids.is_a? Array
+      service_ids = (service_ids.compact_blank - [EMPTY_VALUE]).map(&:to_i)
       member_permission = services_member_permission || member_permissions.build(admin_section: :services)
       member_permission.service_ids = service_ids & existing_service_ids
-    else
+    elsif service_ids.blank?
```

Comment:
This allows us to set `member_permission_service_ids: nil` in the code and `member_permission_service_ids: ""` via API and UI to allow all services.